REQUIREMENTS.md

```
@@ -0,0 +1,101 @@
+## 카드사 식별 번호 구분 규칙
+
+Visa 카드와 Master 카드의 앞 번호의 규칙은 아래로 통일해서 진행한다.
+
+### 💡 카드 브랜드 구분 로직 (Visa / MasterCard)
+
+- Visa: 4로 시작하는 16자리 숫자
+- MasterCard: 51~55로 시작하는 16자리 숫자
+
+---
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)